lib/Runtime/Library/EngineInterfaceObject.h

```diff
@@ -100,10 +100,6 @@ namespace Js
 
         static Var Entry_CallInstanceFunction(RecyclableObject *function, CallInfo callInfo, ...);
 
-#ifdef ENABLE_PROJECTION
-        static Var EntryPromise_EnqueueTask(RecyclableObject *function, CallInfo callInfo, ...);
```

Comment:
Stumbled upon this from the JnDirectFields.h review - this function doesn't exist in chakracore - so deleted.